_automating-workflows/api/create-workflow.md

```diff
@@ -0,0 +1,247 @@
+    # They will be added automatically and included in the stored template
+    # Additional edges may also be added here if required for sequencing
+    edges:
+    - source: create_connector_1
+      dest: register_model_2
+    - source: register_model_2
+      dest: deploy_model_3
+```
+{% include copy-curl.html %}
+
+#### Example Request: Register and deploy a remote model (JSON)
+
+To provide a template in JSON format, specify `Content-Type: application/json` in the request header:
+
+```bash
+curl -XPOST "http://localhost:9200/_plugins/_flow_framework/workflow" -H 'Content-Type: application/json'
+```
+The following JSON template is equivalent to the YAML template in the previous section: 
+
+```json
+{
+  "name": "createconnector-registerremotemodel-deploymodel",
+  "description": "This template creates a connector to a remote model, registers it, and deploys that model",
+  "use_case": "REMOTE_MODEL_DEPLOYMENT",
+  "version": {
+    "template": "1.0.0",
+    "compatibility": [
+      "2.12.0",
+      "3.0.0"
+    ]
+  },
+  "workflows": {
+    "provision": {
+      "nodes": [
+        {
+          "id": "create_connector_1",
+          "type": "create_connector",
+          "user_inputs": {
+            "name": "OpenAI Chat Connector",
+            "description": "The connector to public OpenAI model service for GPT 3.5",
+            "version": "1",
+            "protocol": "http",
+            "parameters": {
+              "endpoint": "api.openai.com",
+              "model": "gpt-3.5-turbo"
+            },
+            "credential": {
+              "openAI_key": "12345"
+            },
+            "actions": [
+              {
+                "action_type": "predict",
+                "method": "POST",
+                "url": "https://${parameters.endpoint}/v1/chat/completions"
+              }
+            ]
+          }
+        },
+        {
+          "id": "register_model_2",
+          "type": "register_remote_model",
+          "previous_node_inputs": {
+            "create_connector_1": "connector_id"
+          },
+          "user_inputs": {
+            "name": "openAI-gpt-3.5-turbo",
+            "function_name": "remote",
+            "description": "test model"
+          }
+        },
+        {
+          "id": "deploy_model_3",
+          "type": "deploy_model",
+          "previous_node_inputs": {
+            "register_model_2": "model_id"
+          }
+        }
+      ],
+      "edges": [
+        {
+          "source": "create_connector_1",
+          "dest": "register_model_2"
+        },
+        {
+          "source": "register_model_2",
+          "dest": "deploy_model_3"
+        }
+      ]
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+#### Example response
+
+OpenSearch responds with the `workflow_id`:
+
+```json
+{
+  "workflow_id" : "8xL8bowB8y25Tqfenm50"
+}
+```
+
+Once you have created a workflow, you can use other workflow APIs with the `workflow_id` .
```

Comment:
```suggestion
Once you have created a workflow, you can use other flow framework APIs with the `workflow_id` .
```

Reply:
@owaiskazi19 @dbwiddis Since I'm generally not referring to "flow framework", I decided to call the overall APIs "workflow APIs". This is the heading structure. What do you think?
![Screenshot 2024-01-25 at 4 04 43 PM](https://github.com/opensearch-project/documentation-website/assets/105444904/47a2bb62-abc4-4518-acc5-452da066e8f2)

Reply:
I'm ok with this structure.